pyproject.toml

```
@@ -0,0 +1,67 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "statick-web"
+authors = [{name = "NIWC Pacific"}]
+description="Statick analysis plugins for Web (css, html, js) files."
+version = "0.2.0"
+readme = "README.md"
+requires-python = ">=3.9"
+license = {text = "CC0-1.0"}
+classifiers = [
+    "License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Topic :: Software Development :: Quality Assurance",
+    "Topic :: Software Development :: Testing",
+    "Typing :: Typed",
+]
+
+dependencies = [
+    "importlib_metadata",
+    "statick@git+https://github.com/tdenewiler/statick@stdlib-plugins",
+]
+
+[tool.setuptools.package-data]
+statick_tool = [
+    "rsc/*",
+    "rsc/.*",
+]
+
+[project.entry-points."statick_tool.plugins.discovery"]
+css = "statick_tool.plugins.discovery.css:CSSDiscoveryPlugin"
+html = "statick_tool.plugins.discovery.html:HTMLDiscoveryPlugin"
+javascript = "statick_tool.plugins.discovery.javascript:JavaScriptDiscoveryPlugin"
+
+[project.entry-points."statick_tool.plugins.tool"]
+eslint = "statick_tool.plugins.tool.eslint:ESLintToolPlugin"
+htmllint = "statick_tool.plugins.tool.htmllint:HTMLLintToolPlugin"
+jshint = "statick_tool.plugins.tool.jshint:JSHintToolPlugin"
+stylelint = "statick_tool.plugins.tool.stylelint:StylelintToolPlugin"
+
+[project.urls]
+"Homepage" = "https://github.com/sscpac/statick-web"
+"Bug Tracker" = "https://github.com/sscpac/statick-web/issues"
+
+[project.optional-dependencies]
+test = [
+    "coverage",
+    "mock",
+    "pytest",
+    "pytest-cov",
+    "statick-md@git+https://github.com/tdenewiler/statick-md@stdlib-plugins",
```

```suggestion
    "statick-md>=1.0.0",
```
once that PR is merged and we have a new release.